common/views/themes/base/transitions.ts

```diff
@@ -0,0 +1,41 @@
+export const transitions = `
+  @keyframes out {
+    from {
+      transform: translateX(0%)
+    }
+
+    to {
+      transform: translateX(100%)
+    }
+  }
+
+  @keyframes in {
+    from {
+      transform: translateX(-100%)
+    }
+
+    to {
+      transform: translateX(0%)
+    }
+  }
+
+  ::view-transition-old(player-previous) {
+    animation: 200ms out ease-in;
+  }
+  ::view-transition-new(player-previous) {
+    animation: 200ms in ease-in;
+  }
+
+  ::view-transition-old(player-next) {
+    animation: 200ms out ease-in reverse;
+  }
+  ::view-transition-new(player-next) {
+    animation: 200ms in ease-in reverse;
+  }
+
+  @media (prefers-reduced-motion: reduce) {
+	  * {
+      view-transition-name: none !important;
+    }
+  }
```

Comment:
Just a note, this looks fine in VSCode, I wonder why the indent on 37 is wonky here
<img width="340" alt="Screenshot 2024-10-02 at 11 54 22" src="https://github.com/user-attachments/assets/1add2053-eba6-45e7-962f-bc3ffc3059a0">

Reply:
To me ! Weiiiird.
<img width="383" alt="Screenshot 2024-10-02 at 12 10 39" src="https://github.com/user-attachments/assets/38a99a5b-b332-4baf-90de-36c374df177a">

Reply:
yikes